internal/controller/monovertexrollout/monovertexrollout_controller_test.go

```diff
@@ -645,40 +646,41 @@ func Test_processExistingMonoVertex_Progressive(t *testing.T) {
 				ctlrcommon.DefaultTestMonoVertexRolloutName + "-1": common.LabelValueUpgradePromoted,
 			},
 		},
-		{
-			name:                           "AnalysisRun failed",
-			newControllerInstanceID:        "1",
-			existingOriginalMonoVertexDef:  &defaultOriginalMonoVertexDef,
-			existingUpgradingMonoVertexDef: defaultUpgradingMonoVertexDef,
-			initialRolloutPhase:            apiv1.PhasePending,
-			initialRolloutNameCount:        2,
-			initialInProgressStrategy:      &progressiveUpgradeStrategy,
-			initialUpgradingChildStatus: &apiv1.UpgradingMonoVertexStatus{
-				UpgradingPipelineTypeStatus: apiv1.UpgradingPipelineTypeStatus{
-					UpgradingChildStatus: apiv1.UpgradingChildStatus{
-						Name:                     ctlrcommon.DefaultTestMonoVertexRolloutName + "-1",
-						BasicAssessmentStartTime: &metav1.Time{Time: time.Now().Add(-1 * time.Minute)},
-						AssessmentResult:         apiv1.AssessmentResultUnknown,
-						InitializationComplete:   true,
-					},
-					Analysis: apiv1.AnalysisStatus{
-						AnalysisRunName: ctlrcommon.DefaultTestMonoVertexRolloutName + "-1",
-						StartTime:       &metav1.Time{Time: time.Now().Add(-45 * time.Second)},
-						EndTime:         &metav1.Time{Time: time.Now().Add(-40 * time.Second)},
-						Phase:           argorolloutsv1.AnalysisPhaseFailed,
-					},
-				},
-			},
-			initialPromotedChildStatus:   defaultPromotedChildStatus,
-			analysisRun:                  true,
-			expectedInProgressStrategy:   apiv1.UpgradeStrategyProgressive,
-			expectedRolloutPhase:         apiv1.PhasePending,
-			expectedProgressiveCondition: metav1.ConditionFalse,
-			expectedMonoVertices: map[string]common.UpgradeState{
-				ctlrcommon.DefaultTestMonoVertexRolloutName + "-0": common.LabelValueUpgradePromoted,
-				ctlrcommon.DefaultTestMonoVertexRolloutName + "-1": common.LabelValueUpgradeInProgress,
-			},
-		},
+		// TODO: re-enable this test once we have fix the time related issues after adding rolling window.
```

Comment:
have you created an issue to track this?

Reply:
https://github.com/numaproj/numaplane/issues/855